docs/extensions/redirects.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: MIT
+
+import json
+import os
+from typing import Dict
+
+from _types import SphinxExtensionMeta
+from sphinx.application import Sphinx
+
+
+def main(app: Sphinx, exception: Exception) -> None:
+    # mapping of html node id (i.e., thing after "#" in URLs) to the same but
+    # prefixed with the right doc name
+    # e.g, api.html#disnake.Thread => api/channels.html#disnake.Thread
+    actual_redirects: Dict[str, str] = {}
+
+    for domainname, domain in app.env.domains.items():
```

Comment:
Why not use `domain = app.env.domains["py"]`?

Reply:
because im not that smart :^

Reply:
Done in [`d8070db`](https://github.com/DisnakeDev/disnake/pull/722/commits/d8070db1f45fdb46fc3264db52d117c2ad5e86f6)